File: app/src/main/java/com/tobikster/medicreminder/data/Reminder.java
package com.tobikster.medicreminder.data;

import org.joda.time.LocalTime;

public class Reminder {
	private String name;
	private LocalTime time;

	public Reminder(String name, LocalTime time) {
		this.name = name;
		this.time = time;
	}

	public String getName() {
		return name;
	}

	public LocalTime getTime() {
		return time;
	}
}

File: app/src/main/java/com/tobikster/medicreminder/data/reminders/Reminder.kt
package com.tobikster.medicreminder.data.reminders

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import org.threeten.bp.LocalTime

@Entity
data class Reminder(
		@PrimaryKey(autoGenerate = true)
		val id: Long,
		val name: String,
		val time: LocalTime
)